pkg/model/dev.go

```diff
@@ -268,6 +270,16 @@ func (dev *Dev) validate() error {
 		return err
 	}
 
+	if !dev.PersistentVolumeEnabled() {
+		if len(dev.Services) > 0 {
+			return fmt.Errorf("'persistentVolume' must be set to true to work with services")
+		}
+
+		if len(dev.Volumes) > 0 {
```

Comment:
Thinking twice about this,  we can enable volumes in this case. The reason would be to mount subpaths of the local folder at a given path in the pod. This is useful to set config files.

Reply:
Although I am finde deactivating them too and only support that use case with persistent volumes

Reply:
I think there's value on the subpath case. I'd rather support as many features as possible without persistence enabled. What's needed for that? Just let the validation pass?

Reply:
Right. Just remove the validation code

Reply:
> The reason would be to mount subpaths of the local folder at a given path in the pod

Reply:
Oh wait, you're talking about the 'volume from local folder' feature right?

Reply:
Right